spec/takeaway_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  it{ is_expected.to respond_to(:view_dishes) }
+
+  it 'displays dishes when view_dishes is called' do
+  expect(subject.view_dishes).to eq({:bacon => 2.50, :cheese => 1.00, :pizza => 5.00, :cake => 4.32, :carrots => 1.20, :pie => 2.00})
```

Comment:
Line is too long. [133/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 0) spaces for indentation.<br>Space inside { missing.<br>Space inside } missing.